causalpy/tests/test_integration_pymc_examples.py

```diff
@@ -191,6 +191,21 @@ def test_sc():
     assert len(result.idata.posterior.coords["draw"]) == sample_kwargs["draws"]
 
 
+@pytest.mark.integration
+def test_sc_input_error():
+    """Test that an error is raised if the data index is not datetime and the
+    treatment time is pd.Timestamp."""
+    with pytest.raises(AssertionError):
```

Comment:
you can use the `match` argument to specify the `AssertionError` string so that is clear which error is expected.

Reply:
I'm assuming this suggestion is moot, given the suggestion below to raise a custom exception?